### PR TITLE
V0.9.5 branch

### PR DESCRIPTION
**Pull request by reedstrm**
packaging fix for static files
Tag created from this branch, BTW)

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -3,5 +3,9 @@ recursive-include cnxpublishing/sql *.sql
 recursive-include cnxpublishing/sql/migrations *.py
 recursive-include cnxpublishing/tests/data *.*
 recursive-include cnxpublishing/views/templates *.*
+recursive-include cnxpublishing/static *.css
+recursive-include cnxpublishing/static/css *.*
+recursive-include cnxpublishing/static/js *.*
+recursive-include cnxpublishing/static/js/vendor *.*
```
Comment (pumazi) [inline MANIFEST.in]: `cnxpublishing/static/** *.*` single line for this.

**Comment by reedstrm** [comment]
There seem to be race conditions now in the publishing tests - hammering travis w/ restarts gets them to pass, occasionally.

**Comment by reedstrm** [comment]
Yes, I got the PR to pass by restarting it 3-4 times.

**Comment by pumazi** [comment]
I've been able to get the push tests to pass after rebasing (on a new branch). We'll take this as is though since it was where the tag was created. Furthermore, the pr tests pass, because the merge commit is present in the pr test run, which means the #202 fix is present.

**Pull request merged**